src/procedural.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace procedural;
```

Comment:
yes, we use same directory, it is strange to have different namespace in same directory.